src/ServerMain.cpp

```diff
@@ -104,6 +105,13 @@ int main(int argc, char** argv) {
       po::bool_switch(&onlyPsoAndPosPermutations),
       "Only load the PSO and POS permutations. This disables queries with "
       "predicate variables.");
+  add("default-query-timeout,s",
+      po::value<NonNegative>()->default_value(10)->notifier(
```

Comment:
Maybe also something like `30` would also be reasonable, but typically this should always be set:)